olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -240,31 +240,44 @@ std::shared_ptr<http::NetworkRequest> OlpClient::CreateRequest(
 }
 
 NetworkAsyncCallback GetRetryCallback(
-    int current_try, int current_backdown_period, const RetrySettings& settings,
+    int current_try, int current_backdown_period,
+    long long accumulated_wait_time, const RetrySettings& settings,
```

Comment:
Please use int64_t or uint64_t instead of plain old data types.
Same for all other occurrences below.

Reply:
I used `std::chrono::milliseconds` instead.